uwsgi/admin.wsgi

```diff
@@ -15,6 +15,7 @@ DOMAIN_WHITELIST = {
     "clients2.googleusercontent.com": ("Widevine",),
     "redirector.gvt1.com": ("Widevine",),
     "ftp.mozilla.org": ("SystemAddons",),
+    "bucketlister-delivery.stage.mozaws.net": ("Fennec-tcmigration", "Firefox-tcmigration"),
```

Comment:
Since @mostlygeek has offered to add an env variable for us, we can do better! Instead of this, add this new domain in an 'if os.environ.get("XXXXXXX") == "stage"' block. Since it will only apply to stage, you don't need to use fake product names either.